Gemfile.lock

```diff
@@ -206,10 +214,11 @@ GEM
     websocket-extensions (0.1.3)
 
 PLATFORMS
-  ruby
+  x86-mingw32
```

Comment:
Just a note that this might break anyone's `Gemfile.lock` if they use a Mac. They might have to remove this file and generate a new `Gemfile.lock`. It might make more sense for Annie to not check this in (since she's the only one with a Windows) and have the correct version of `Gemfile.lock` checked in next time it's updated by a Mac. Not sure what approach would work best here.

Reply:
just double checking: does it break if you checkout this branch and run bundle install?

Reply:
Yes, it breaks on my system when I try to run bundle install. Also, could you please help specify a version of pry-byebug and pry? I think I forgot that when I put those gems in.